receiver/kafkareceiver/README.md

```diff
@@ -87,11 +87,51 @@ The following settings can be optionally configured:
   - `after`: (default = false) If true, the messages are marked after the pipeline execution
   - `on_error`: (default = false) If false, only the successfully processed messages are marked
     **Note: this can block the entire partition in case a message processing returns a permanent error**
-
+- `header_extraction`:
+  - `extract_headers` (default = false): Allows user to attach header fields to resource attributes in otel piepline
```

Comment:
This seems  redundant, the extraction can be controlled by `headers`  option

Reply:
do other collector components have a similar config - e.g. enable field?

Reply:
Yeah, I saw `healtcheckextension` has two fields, one to enable and one for the interval. I kept two fields just in case. 
Consider the user has a large list of headers to extract and he/she wants to frequently enable/disable to test it out. It can be done quickly with the `enable` field.

But you're right, it can also be controlled with the `headers` option.